_dashboards/management/scheduled-query-acceleration.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: default
+title: Scheduled Query Acceleration
+parent: Data sources
+nav_order: 18
+has_children: false
+---
+
+# Scheduled Query Acceleration
+Introduced 2.17
+{: .label .label-purple }
+
+Scheduled Query Acceleration (SQA) is designed to optimize direct queries from OpenSearch to Amazon Simple Storage Service (Amazon S3). It addresses issues often faced when managing and refreshing indexes, views, and data in an automated way. 
```

Comment:
> OpenSearch to Amazon Simple Storage Service (Amazon S3)

This direct query is not only limited to s3, let's use "external data sources" on this page for consistency with other pages